boefjes/boefjes/katalogus/local_repository.py

```diff
@@ -158,3 +160,25 @@ def _normalizer_to_plugin(normalizer: NormalizerResource) -> Normalizer:
 
 def get_local_repository():
     return LocalPluginRepository(BOEFJES_DIR)
+
+
+def get_runnable_hash(path: Path) -> str:
+    """Returns sha256(sha256(file1) + sha256(file2) + ...) of all files in the given path."""
+
+    file_hashes = []
+
+    for file in path.iterdir():
```

Comment:
`iterdir` doesn't iterate in a stable order, which can cause different hashes between runs.

Reply:
Indeed. You could sort this iteration which should give a pretty stable outcome

Reply:
Ah of course! Completely overlooked the sorting order stability.